aamt_parse2.pm

```diff
@@ -308,12 +556,13 @@ sub pars_CreateReadinessReport
             {
                 ilog_print(1,"\n\nPublishing failed with response code: $rCode\n");            
                 my $strYesOrNo ="";
-                while($strYesOrNo!~/^\s*[YynN]\s*$/)
+#                while($strYesOrNo!~/^\s*[YynN]\s*$/)
                 {
-                    ilog_printf(1, "    Would you like to retry uploading the readiness report? (Y/N):");
```

Comment:
You need to add a cmd line flag for running in unattended/non-interactive mode. If this flag is true, then you can skip retries, if it is false, then keep the current logic of asking the user for input.